src/layer_shims.cpp

```diff
@@ -432,12 +432,10 @@ void recordGetActionStateFloat(const XrActionStateGetInfo *getInfo, XrActionStat
     auto isActive = state->isActive;
     auto lastChanged = state->lastChangeTime;
 
-    // we don't need all values. especially if there are long periods that noting happens. check if we need to log
```

Comment:
Any reason these comments are deleted? If they are outdated, could you replace, not delete? :)